test/decl/operator/Inputs/power_operator_declaration/PowerOperatorDeclaration.h

```diff
@@ -0,0 +1,2 @@
+@interface PowerOperator : NSObject
```

Comment:
I'm not sure I understand: Why is this Obj-C module called a "power operator declaration"? It doesn't seem to declare a power operator anywhere. Do you mean simply that it declares `pow` (not the operator)? Can we just test by importing the C standard library directly instead?

Reply:
Thanks I removed it and fixed it to test by importing `cfuncs` instead :)